src/app/pages/SearchResultsPage/ResultsElsewhere.tsx

```diff
@@ -0,0 +1,148 @@
+import { FC, useState } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { styled } from '@mui/material/styles'
+import { getThemesForNetworks } from '../../../styles/theme'
+import Box from '@mui/material/Box'
+import { SearchQueries } from './hooks'
+import ZoomIn from '@mui/icons-material/ZoomIn'
+import ZoomOut from '@mui/icons-material/ZoomOut'
+import { getKeyForScope, getScopeForKey, SearchScope } from '../../../types/searchScope'
+import { getNetworkNames, Network } from '../../../types/network'
+import { RouteUtils } from '../../utils/route-utils'
+import { ResultsFilteredThemed } from './ResultsFilteredThemed'
+
+const NotificationBox = styled(Box)(({ theme }) => ({
+  // TODO: this is probably not fully correct.
+  marginTop: 30,
+  marginBottom: 30,
+  fontSize: 14,
+  marginLeft: '10%',
+  marginRight: '10%',
+
+  boxSizing: 'border-box',
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  alignItems: 'center',
+  padding: '5px 10px',
+  gap: 10,
+
+  height: 50,
+
+  background: theme.palette.layout.notificationBackground,
+  border: `2px solid ${theme.palette.layout.darkBorder}`,
+  color: theme.palette.layout.notificationText,
+
+  borderRadius: 50,
+}))
+
+export const ResultsElsewhere: FC<{
+  wantedScope: SearchScope
+  resultsInScopes: Record<string, number>
+  searchQueries: SearchQueries
+  roseFiatValue: number | undefined
+}> = ({ wantedScope, resultsInScopes, searchQueries, roseFiatValue }) => {
+  const { t } = useTranslation()
+  const wantedScopeKey = getKeyForScope(wantedScope)
+  const alsoHasWantedResults = !!resultsInScopes[wantedScopeKey]
+  const otherScopesKeys = Object.keys(resultsInScopes)
+    .filter(key => !!resultsInScopes[key])
+    .filter(scopeKey => scopeKey !== wantedScopeKey)
+  const otherScopes = otherScopesKeys.map(getScopeForKey)
+  const hasResultsOnDifferentParatimes = otherScopes.some(
+    scope => scope.network === wantedScope.network && scope.layer !== wantedScope.layer,
+  )
+  const hasResultsOnDifferentNetworks = otherScopes.some(scope => scope.network !== wantedScope.network)
+  const hasAllKindsOfResults = hasResultsOnDifferentParatimes && hasResultsOnDifferentNetworks
+  const location = t(
+    hasAllKindsOfResults
+      ? 'search.otherResults.otherNetworksAndParatimes'
+      : hasResultsOnDifferentNetworks
+      ? 'search.otherResults.otherNetworks'
+      : 'search.otherResults.otherParatimes',
+  )
+
+  const totalNumberOfResults = otherScopesKeys.reduce(
+    (partialSum, scopeKey) => partialSum + resultsInScopes[scopeKey],
+    0,
+  )
+
```

Comment:
there has to be a more readable solution

Reply:
Yes, and I plan to implement it after https://github.com/oasisprotocol/explorer/pull/453/commits/741ed2801434b82df25d1b8920d098cfe242b407 lands. Originally it has been implemented as part of a different PR, but maybe I will pull it into this PR...

Reply:
Actually, I would like to ask you to review #453 first, or at least the aforementioned commit. If we are in agreement on that change, then I will rework this logic on top of that code.

Reply:
Please check out the same section in the current code; I think it has been significantly improved.